packages/core/src/result-transformers.ts

```diff
@@ -74,9 +74,6 @@ class ResultTransformers {
    * const { keys, records, summary } = await driver.executeQuery('CREATE (p:Person{ name: $name }) RETURN p', { name: 'Person1'})
    *
    * @returns {ResultTransformer<EagerResult<Entries>>} The result transformer
-   * @experimental this is a preview
-   * @since 5.22.0
```

Comment:
Is this info no longer valuable? (more `@since` removals in patch & `#summary` didn't have a `@since` to begin with, won't comment again)